From a7e4396c25db230e871695790699aea28810ace9 Mon Sep 17 00:00:00 2001
From: Vladimir Barinov <vladimir.barinov@cogentembedded.com>
Date: Fri, 6 Mar 2020 18:30:23 +0300
Subject: [PATCH] media: i2c: ov10640: enable soft reset

This enables soft reset for imager to avoid
unexpected behavoir: f.e. if power rails were not
correctly reset

Signed-off-by: Vladimir Barinov <vladimir.barinov@cogentembedded.com>
---
 drivers/media/i2c/soc_camera/ov10640_r1d.h | 4 ++--
 drivers/media/i2c/soc_camera/ov10640_r1e.h | 4 ++--
 drivers/media/i2c/soc_camera/ov10640_r1f.h | 4 ++--
 3 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/drivers/media/i2c/soc_camera/ov10640_r1d.h b/drivers/media/i2c/soc_camera/ov10640_r1d.h
index 6f29b01..ac31402 100644
--- a/drivers/media/i2c/soc_camera/ov10640_r1d.h
+++ b/drivers/media/i2c/soc_camera/ov10640_r1d.h
@@ -11,8 +11,8 @@
 
 /* DVP_1280x960_COMB12_raw 30fps */
 static const struct ov10640_reg ov10640_regs_wizard_r1d[] = {
-//{0x3013, 0x01},
-//{OV10640_DELAY, 10},
+{0x3013, 0x01},
+{OV10640_DELAY, 20},
 {0x328a, 0x01},
 {0x313f, 0x80},
 {0x3132, 0x24},
diff --git a/drivers/media/i2c/soc_camera/ov10640_r1e.h b/drivers/media/i2c/soc_camera/ov10640_r1e.h
index ba3c636..adda111 100644
--- a/drivers/media/i2c/soc_camera/ov10640_r1e.h
+++ b/drivers/media/i2c/soc_camera/ov10640_r1e.h
@@ -11,8 +11,8 @@
 
 /* DVP_1280x1080_COMB12_raw 60fps */
 static const struct ov10640_reg ov10640_regs_wizard_r1e[] = {
-//{0x3013, 0x01},
-//{OV10640_DELAY, 10},
+{0x3013, 0x01},
+{OV10640_DELAY, 20},
 {0x328a, 0x11},
 {0x313f, 0x80},
 {0x3132, 0x24},
diff --git a/drivers/media/i2c/soc_camera/ov10640_r1f.h b/drivers/media/i2c/soc_camera/ov10640_r1f.h
index 3f9b3f5..8911732 100644
--- a/drivers/media/i2c/soc_camera/ov10640_r1f.h
+++ b/drivers/media/i2c/soc_camera/ov10640_r1f.h
@@ -11,8 +11,8 @@
 
 /* DVP_1280x1080_COMB12_raw 60fps */
 static const struct ov10640_reg ov10640_regs_wizard_r1f[] = {
-//{0x3013, 0x01},
-//{OV10640_DELAY, 10},
+{0x3013, 0x01},
+{OV10640_DELAY, 20},
 {0x31be, 0x01},
 {0x3133, 0xb7},
 {0x3134, 0xca},
-- 
2.7.4

